### PR TITLE
Make more strings translatable

### PR DESCRIPTION
**Pull request by vnctdj**

### CODE REVIEW

File: UI/RemoteISOScreen.cpp [hunk 1]
```diff
@@ -397,7 +398,7 @@ void RemoteISOScreen::CreateViews() {
 		rightColumnItems->Add(new Choice(sy->T("Share Games (Server)")))->OnClick.Handle(this, &RemoteISOScreen::HandleStartServer);
 		browseChoice->SetEnabled(true);
 	}
-	Choice *settingsChoice = new Choice(sy->T("Settings"));
+	Choice *settingsChoice = new Choice(pa->T("Settings"));
```
Comment (unknownbrackets) [inline UI/RemoteISOScreen.cpp]: Not that System was really right, but Pause doesn't seem related.  I hate reusing a word like "Settings" in different contexts... I guess it's hopefully the same here.

-[Unknown]

File: UI/RemoteISOScreen.cpp [hunk 2]
```diff
@@ -598,8 +600,8 @@ RemoteISOBrowseScreen::RemoteISOBrowseScreen(const std::vector<std::string> &gam
 void RemoteISOBrowseScreen::CreateViews() {
 	bool vertical = UseVerticalLayout();
 
-	I18NCategory *mm = GetI18NCategory("MainMenu");
 	I18NCategory *di = GetI18NCategory("Dialog");
+	I18NCategory *n = GetI18NCategory("Networking");
```
Comment (unknownbrackets) [inline UI/RemoteISOScreen.cpp]: This definitely is not related to "networking", and if someone translated it thinking it was, they'd probably be translating it wrong.

Even if you disable game netplay, this functionality still works.  This functionality is more for when you want to play a game on your Android tablet, but you want to use the ISO that is on your PC.  It has nothing to do with netplay, and I don't want any of the UI to suggest otherwise - that would only confuse people.

-[Unknown]

Comment (vnctdj) [inline UI/RemoteISOScreen.cpp]: Ah...
I used "Networking" as a general purpose network-related category.
Though I do understand the difference between the functionality you explain and netplay.

Therefore I think we should make the distinction more obvious.
Why not having 2 categories ? :
 - "Networking" --> remote ISO feature
 - "NetPlay"/"Adhoc"/"Multiplayer" (which name ?) --> netplay

File: UI/RemoteISOScreen.cpp (hunk 1 above)
Comment (vnctdj) [inline UI/RemoteISOScreen.cpp]: Yes indeed "Pause" has nothing to do here...
See my reply to your other comment below for an idea of a better category.

File: UI/RemoteISOScreen.cpp (hunk 2 above)
Comment (unknownbrackets) [inline UI/RemoteISOScreen.cpp]: Well, these are the existing ones:

https://github.com/hrydgard/ppsspp/blob/ec7e6342151e9303eeafcf54ba79cfb4c8f757e1/UI/GameSettingsScreen.cpp#L613-L626
https://github.com/hrydgard/ppsspp/blob/d30116b8c3f508a1696d7da39e5ab5dd874ad524/Core/HLE/proAdhoc.cpp#L1461

And then here for the other type:
https://github.com/hrydgard/ppsspp/blob/c35461b75cceadcdea1709a9866600990227276f/UI/RemoteISOScreen.cpp#L678-L687

In settings, the heading is "Networking" so I'd vote on changing the remote iso feature - maybe even `[RemoteISO]` and `rs`?

-[Unknown]

Comment (vnctdj) [inline UI/RemoteISOScreen.cpp]: This makes sense :)
But why "rs" for the pointer name ? I think "ri" looks better to represent "RemoteISO".

Comment (unknownbrackets) [inline UI/RemoteISOScreen.cpp]: Oops, sure.  Not sure now, rs was probably a typo.

-[Unknown]

Comment (vnctdj) [inline UI/RemoteISOScreen.cpp]: Oh ok :)
I will push a new commit soon.

**Review by unknownbrackets — APPROVED** [review]

**Comment by vnctdj** [comment]
Thanks for the review and for merging :)
